### PR TITLE
fix: dev:Configuring Sonar to show code quality

### PR DESCRIPTION
**Pull request by flopezag**
### REVIEWERS

@chemaper , @jframos 

### DESCRIPTION
- Configure sonar project properties.

### TESTS
Tests in the Jenkins environment.

### CODE REVIEW

**Comment by chemaper** [comment]
LGTM

File: sonar-project.properties [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+# sonar.properties
+#
+# The configuration information to be taken in order to generate the information
+# in Sonar. The project key will be com.telefonica.iot:skuld
+#
+# __author__ = fla
+
+sonar.projectName=Skuld
+sonar.projectKey=com.telefonica.iot:skuld
+sonar.sources=./skuld
```
Comment (jframos) [inline sonar-project.properties]: It this source path correct? Or should it be like this ?
  
     sonar.sources=./

Comment (flopezag) [inline sonar-project.properties]: You are right but we need to extend the content of the sonar-project.properties

**Comment by jframos** [comment]
LGTM

**Pull request merged**